git-players.text

```diff
@@ -1,6 +1,13 @@
 <<record
-player_name: THS
-player_display_name: THSPower
+player_name: MTS
+player_display_name: MTSEnergy
 player_github: https://github.com/thspower
-player_favorite_thing: Saving the world
```

Comment:
Hey I dont think the players already in the file should be changed.

Reply:
I came back with the player who had been removed!